.pre-commit-config.yaml

```diff
@@ -2,12 +2,12 @@ default_language_version:
   python: python3
 repos:
   - repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v3.2.0
+    rev: v3.3.0
     hooks:
       - id: trailing-whitespace
       - id: end-of-file-fixer
       - id: fix-encoding-pragma
-        exclude: ^arrow/_version.py
```

Comment:
This is meant to be gone?

Reply:
Yeah that was there since we didn't want to put the encoding pragma in `_version.py`, but in a Python 3.6+ codebase, the encoding pragma is not needed at all.